phc-rails/config/routes.rb

```diff
@@ -7,6 +7,7 @@
       get "search", to: "accounts#search"
       post "create", to: "accounts#create"
       get "check", to: "accounts#check"
+      get 'accounts/:sf_id', to: 'accounts#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.